frameworks/kafka/src/main/dist/server.properties.mustache

```diff
@@ -27,9 +27,28 @@ broker.id={{POD_INSTANCE_INDEX}}
 #     listeners = security_protocol://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
+{{^KAFKA_ENABLE_TLS}}
```

Comment:
FWIW naming aside, svc.yml templating is based on the scheduler's environment, while templating in this config file is based on the task's environment